### PR TITLE
[_]: feat/test-password-strength

### PR DESCRIPTION
**Pull request by AlexMenor**

### CODE REVIEW

File: test/auth/testPasswordStrength.test.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    if (result.valid === false) {
+      expect(result.reason).toBe('NOT_COMPLEX_ENOUGH');
+    }
+  });
+
+  it('should consider invalid a password with lowercase, uppercase and digits that looks like his name', () => {
+    const result = testPasswordStrength('PassWord4', 'expert_word_user');
+    expect(result.valid).toBe(true);
+
+    if (result.valid) {
```
Comment (sg-gs) [inline test/auth/testPasswordStrength.test.ts:37]: This check is required? The test will fail if `result.valid` is not true

File: test/auth/testPasswordStrength.test.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      expect(result.strength).toBe('medium');
+    }
+  });
+
+  it('should consider invalid a password with lowercase, uppercase and symbols that looks like his name', () => {
+    const result = testPasswordStrength('PassWord#', 'expert_word_user');
+    expect(result.valid).toBe(true);
+
+    if (result.valid) {
```
Comment (sg-gs) [inline test/auth/testPasswordStrength.test.ts:46]: Same here

File: test/auth/testPasswordStrength.test.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      expect(result.strength).toBe('medium');
+    }
+  });
+
+  it('should consider invalid a password with lowercase, uppercase, digits and symbols strong', () => {
+    const result = testPasswordStrength('PassWord#4', 'expert_word_user');
+    expect(result.valid).toBe(true);
+
+    if (result.valid) {
```
Comment (sg-gs) [inline test/auth/testPasswordStrength.test.ts:55]: And here

File: test/auth/testPasswordStrength.test.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import { auth } from '../../src';
+
+const { testPasswordStrength } = auth;
+
+describe('testPasswordStrength tests', () => {
+  it('should consider invalid a password with less than 8 characters', () => {
+    const result = testPasswordStrength('abcd', 'whatever');
+    expect(result.valid).toBe(false);
+
+    if (result.valid === false) {
```
Comment (sg-gs) [inline test/auth/testPasswordStrength.test.ts:10]: Same with the `false` one

**Review by sg-gs — APPROVED** [review]

File: test/auth/testPasswordStrength.test.ts (hunk 4 above)
Comment (AlexMenor) [reply test/auth/testPasswordStrength.test.ts:10]: required by typescript @sg-gs

**Pull request merged**